department-of-veterans-affairs/GIBCT-Data-Service
# Added programs.csv to GIDS

## Description
As an EDU team member, I need the ability to upload program data to GIDS so that it can be used to display information in the Comparison Tool

## Testing done

- Open GIDS
- Navigate to the programs and click export
- Add a row in the programs.csv
- Upload the updated programs.csv in the GIDS tool
- Check the POSTGRE database to confirm changes
- Download export CSV and confirm changes

## Screenshots
![Screen Shot 2019-08-13 at 10 32 52 AM](https://user-images.githubusercontent.com/50601724/62955229-b1723a80-bdbe-11e9-8d6c-57f3eafb8b1c.png)
![Screen Shot 2019-08-13 at 10 32 19 AM](https://user-images.githubusercontent.com/50601724/62955236-b3d49480-bdbe-11e9-9fd8-0f2060bdb312.png)
![Screen Shot 2019-08-13 at 10 31 00 AM](https://user-images.githubusercontent.com/50601724/62955249-b6cf8500-bdbe-11e9-809c-4f5c9173006e.png)


## Acceptance criteria
- [x] A user can upload a .csv file containing program data matching the format outlined in Supporting Artifact # 1 into the GIDS Tool.
- [x] A user can export the most recently uploaded program data from the GIDS Tool.


## Definition of done
